webrender_api/src/api.rs

```diff
@@ -884,8 +886,9 @@ pub struct DynamicProperties {
 
 pub trait RenderNotifier: Send {
     fn clone(&self) -> Box<RenderNotifier>;
-    fn new_frame_ready(&self);
-    fn new_scroll_frame_ready(&self, composite_needed: bool);
+    fn wakeup(&self);
+    fn new_document_ready(&self, DocumentId);
```

Comment:
While you are modifying this interface, maybe it'd be a good time to merge `new_document_ready` and `new_scroll_document_ready` into a single one that has enough parameters (id, did_scroll, composite_needed) to avoid contortions like in #1963 when the scrolled and non-scrolled paths are merged.

Reply:
Agreed.